src/components/savedAccounts/savedAccounts.js

```diff
@@ -102,9 +114,12 @@ class SavedAccounts extends React.Component {
             key={account.publicKey + account.network}
             onClick={ switchAccount.bind(null, account)} >
               {(account.passphrase ?
-                <strong className={styles.unlocked}>
+                <strong
+                  className={`unlocked ${styles.unlocked}`}
+                  onClick={this.handleRemovePassphrase.bind(this, account)}
+                >
                   <FontIcon value='unlocked' />
-                  {t('Unlocked')}
+                  {t('Lock ID')}
```

Comment:
There is one new requirement. When clicked, "Lock ID" should change to "Your ID is now secured!" and disappear only after ~3 seconds.